airflow/plugins_manager.py

```diff
@@ -120,6 +120,9 @@ def is_valid_plugin(plugin_obj, existing_plugins):
 
 norm_pattern = re.compile(r'[/|.]')
 
+if settings.PLUGINS_FOLDER is None:
+    raise AirflowPluginException("Plugins folder is not set")
```

Comment:
Is this ever possible to hit?

Reply:
I think I hit it once - but it could be due to misconfiguration or error why generating settings. I don't think there is harm in keeping it :). It's more of a sanity check.

Reply:
Fair,yeah.